yb-voyager/src/queryparser/helpers.go

```diff
@@ -115,49 +119,22 @@ func GetStringValueFromNode(nodeMsg protoreflect.Message) string {
 	}
 }
 
-// extractStringField safely extracts a string field from a node
-// Sample example:: {column_ref:{fields:{string:{sval:"s"}}  fields:{string:{sval:"tableoid"}}  location:7}
-func extractStringField(node protoreflect.Message, fieldName string) string {
-	strField := node.Descriptor().Fields().ByName(protoreflect.Name(fieldName))
-	if strField == nil || !node.Has(strField) {
-		return ""
-	}
-	return node.Get(strField).String()
-}
-
-// extractAConstString extracts the string from an 'A_Const' node's 'sval' field
+// getStringFromAConstMsg extracts the string from an 'A_Const' node's 'sval' field
 // Sample example:: rowexpr:{a_const:{sval:{sval:"//Product"}  location:124}}
-func extractAConstString(aConstMsg protoreflect.Message) string {
-	// Extract the 'sval' field from 'A_Const'
-	svalField := aConstMsg.Descriptor().Fields().ByName("sval")
-	if svalField == nil || !aConstMsg.Has(svalField) {
-		return ""
-	}
-
-	svalMsg := aConstMsg.Get(svalField).Message()
-	if svalMsg == nil || !svalMsg.IsValid() {
-		return ""
-	}
-
-	// Ensure svalMsg is of type 'pg_query.String'
-	if svalMsg.Descriptor().FullName() != "pg_query.String" {
+func getStringFromAConstMsg(aConstMsg protoreflect.Message) string {
+	svalMsg := GetMessageField(aConstMsg, "sval")
+	if svalMsg == nil {
 		return ""
```

Comment:
I'm a little apprehensive of returning empty strings. It feelsl like we should also be returning an error to indicate that we could not find the string... but looks like there are a lot of uses here. I will think more about this later and get back.